src/config.rs

```diff
@@ -32,8 +35,6 @@ pub struct CliArgs {
     pub no_panel: bool,
     /// Window settings for the initial winit window
     pub window_attributes: WindowAttributes,
-    /// Port number to start a server to listen to remote Firefox devtools connections. 0 for random port.
-    pub devtools_port: Option<u16>,
```

Comment:
Why is this removal? Servo changed `devtools_port` from `Opts` to `Preferences`, not removed

Reply:
With corresponding settings moved to Preference means we have to consciously have a new component maintaining preference if we want to keep these.

Reply:
> With corresponding settings moved to Preference means we have to consciously have a new component maintaining preference if we want to keep these.

I'm not sure what do you mean, can't we just change

```rust
servo_config::prefs::set(Preferences::default());
```

to

```rust
servo_config::prefs::set(Preferences {
    devtools_server_port: ...,
    devtools_server_enabled: ...,
    ..Default::default()
});
```

?

Reply:
Sure, we can just add it back like that for now. It is just removed for the sake of updating servo and the fact that we currently don't have preference differences from servo.